tutorials/server/configure-ssh-2fa/configure-ssh-2fa.md

```diff
@@ -0,0 +1,129 @@
+---
+id: configure-ssh-2fa
+summary: Configure SSH to use two factor authentication
+categories: server
+tags: tutorial,ssh,security,ubuntu,terminal
+difficulty: 2
+status: Published
+published: 2017-12-04
+author: Marcin Mikołajczak <me@m4sk.in>
+
+---
+
+# Configure SSH to use two factor authentication
+Duration: 1:00
+
+## Overview
+
+SSH, the secure shell, is often used to access remote Linux systems. Because we often use it to connect with computer containing important data, it's recommended to add another security layer. Here comes the two factor authentication (*2FA*).
+
+### Requirements
+
+All you need is a PC running Ubuntu and configured SSH connection.
+
+  - You should understand the danger of stolen passwords.
+  - You don't need to know what two factor authentication is and how it works.
+
+
+## What is two factor authentication
+Duration: 1:00
+
+Multi factor authentication is a method of confirming your identity using at least two different was of authentication. The most common and easiest to implement example of two factor authentication uses combination of **passphrase** (often called *password*, which implies that you should just use single word to log in) and one-time-passcode generated by special mobile app.
+
+We will use Google Authenticator app available for Android (in [Play Store]) and iOS (in [iTunes]) to generate authentication codes.
```

Comment:
"We will use **the** Google Authenticator app available for Android (in **the** [Play Store]) and iOS (in [iTunes]) to generate authentication codes."

Maybe one more sentence: "If you are using another phone OS or authentication device, you can follow and adapt this tutorial, as the basic principles of 2FA still apply".